pbjson-build/src/descriptor.rs

```diff
@@ -68,7 +70,7 @@ impl TypeName {
 
     pub fn to_snake_case(&self) -> String {
```

Comment:
Perhaps we should rename this to `to_snake_case_ident` or something, to make clearer that it performs escaping